app/views/maps/show.html.erb

```diff
@@ -103,6 +103,12 @@
       <%= render partial: 'front_ui/maps', locals: { maps: @maps } %>
   </div>
 
+  <div class="all-maps light-blue">
+      <br>
+      <h3 class="text-center"> Exports </h3>
+      <%= render partial: 'maps/cloud_exports', locals: { exports: @map.exports }%>
+  </div>
+
  <script>
     var map
```

Comment:
The `test/system/images_test.rb` failure of `map._layers` doesn't make sense unless this line was interrupted. I'm restarting Travis just to check. 

https://travis-ci.org/publiclab/mapknitter/jobs/642090297